### DISCUSSION TITLE
Linux Support for AppImage and/or Flatpak

### DISCUSSION THREAD

**Discussion by robke96** [Tyrrrz/YoutubeDownloader#542] [Q&A]
I was wondering if there are any plans to make it available as an [AppImage](https://appimage.org/) or [Flatpak](https://flatpak.org/)?

**Comment by Tyrrrz** [comment]
I thought it should already work pretty much on all linux distros out of the box

**Comment by tech4teachers** [comment]
Agreed, this doesn't seem to compile for Debian  from the command line or have any instructions, it is just a pile of DLL's, I was thinking a .deb would be more useful but an appimage or flatpak would be great. I must be missing a dependency but I can't figure out which one.  Really appreciate the tool on Windows, but we are switching school to debian after a kerfuffle over TPM2.0 forcing too much hardware cost to upgrade for Windows 11 reliance.
